packages/cosmic-proto/src/helpers.ts

```diff
@@ -34,3 +41,11 @@ export const typedJson = <T extends keyof Proto3Shape>(
     ...obj,
   } as TypedJson<T>;
 };
+
+// TODO make codegen toJSON() return these instead of unknown
```

Comment:
We will also need this for `ResponseQuery.toJSON()` (`/tendermint/abci/types.js`). Any chance it can fit in this PR?

Reply:
that would require patching the codegen. out of scope, thus the TODO